ops/pebble.py

```diff
@@ -1442,10 +1471,17 @@ def __init__(
         failures: int = 0,
         threshold: int = 0,
         change_id: Optional[ChangeID] = None,
+        startup: Union[CheckStartup, str] = CheckStartup.ENABLED,
     ):
         self.name = name
         self.level = level
-        self.status = status
+        self.startup = startup
+        if change_id:
+            self.status = status
+        else:
+            # No change ID means the check is inactive, or that the user has an
+            # old version of Pebble. We assume the former.
```

Comment:
Hmm, I wonder if we can avoid this. If the `startup` field is present, we know it's a newer version of Pebble. Can we use that fact?

Reply:
Ah nice suggestion, thanks!